lewis/examples/modbus_device/__init__.py

```diff
@@ -40,3 +40,6 @@ class ModbusInterface(ModbusAdapter):
     co = di
     ir = ModbusBasicDataBank(0)
     hr = ir
+
+
+framework_version = '>=1.0.3'
```

Comment:
This should probably be reduced to `1.0.2` for now, like the others, and the ranging removed.